Source/JavaScriptCore/dfg/DFGStoreBarrierInsertionPhase.cpp

```diff
@@ -431,17 +435,50 @@ class StoreBarrierInsertionPhase : public Phase {
                 // So this analysis must mark an allocation that escapes to the heap as being part of the primordial
                 // epoch.
 
-                ClobberSet writes = writeSet(m_graph, m_node);
-                if (writes.overlaps(Heap) || writes.overlaps(Stack)) {
-                    auto escape = [] (Node* node) {
+                auto readFunc = [&] (const AbstractHeap& heap) {
+                    if (heap.kind() == Stack) {
```

Comment:
Don't we need to have `read(World)` support?

Reply:
World will overlap with `heap.overlaps(Heap)` below. Maybe I can make the code a bit more obvious, though.

Reply:
sorry, I thought this comment was about some other code. You're right, let me fix this.